### ISSUE TITLE
Document thumbnails containing images with empty alt tags cause accessibility issues.

### ISSUE THREAD

**Issue by edwardhorsford** [alphagov/whitehall#5896]
Raised on the cross gov accessibility mailing list.

Image thumbnails ([example here](https://www.gov.uk/government/publications/covid-19-winter-plan)) appear to be an anchor wrapping an image asset. The asset has an empty `alt` tag. This is causing some screen readers to read out aspects of the url of the image instead as there's no accessible name.

---

I wonder if either the markup should be changed so there's just one link that wraps everything, or else whether the link in the thumbnail should be hidden from screen readers entirely.

## Original message from the mailing list:

For example, from this page, I've pasted below the speech buffer from my screenreader so you can see exactly what I hear for the "Documents" h2.

heading level 2‑ Documents
Link‑ 937529/COVID-19_Winter_Plan
 
heading level 3‑ Link‑ COVID-19 Winter Plan
PDF, 3.16MB, 64 pages
This file may not be suitable for users of assistive technology.
Request an accessible format.
Button‑ collapsed‑
Link‑ covid-19-winter-plan/covid-19-winter-plan
 
heading level 3‑ Link‑ COVID-19 Winter Plan
HTML
Link‑ 937528/COVID-19_Winter_Plan_Large_Print
 
heading level 3‑ Link‑ COVID-19 Winter Plan - large print
PDF, 3.3MB, 112 pages
This file may not be suitable for users of assistive technology.
Request an accessible format.
Button‑ collapsed‑
Link‑ covid-19-winter-plan/covid-19-winter-plan-summary
 
heading level 3‑ Link‑ COVID-19 Winter Plan - summary 
HTML
heading level 2‑ Details
The UK Government’s COVID-19 Winter Plan presents a programme for suppressing the virus, protecting the NHS and the vulnerable, keeping education and the
economy going and providing a route back to normality. The plan sets out the Government’s approach to ending the national restrictions on 2 December.
Published 23 November 2020
Last updated 23 November 2020
+ show all updates
Button‑ collapsed‑

.............

 

It doesn't happen on all pages. For instance, not on this one, but it does on this one.

**Comment by edwardhorsford** [comment]
Resolved in [govuk-fronend #1923](https://github.com/alphagov/government-frontend/pull/1923)
